### PR TITLE
Add log statements to capture more data about update events. See va.g…

### PR DESCRIPTION
**Pull request by joeniquette**
Add log statements to capture more data about update events. See va.gov-team 25153.

## Description of change
FW&A team would like to have the ICN added to events which a logged in user updates their direct deposit information or updates their contact information.

## Original issue(s)
https://app.zenhub.com/workspaces/vsp-identity-5f5bab705a94c9001ba33734/issues/department-of-veterans-affairs/va.gov-team/25153

## Things to know about this PR

Testing will include the use of each controller on staging to ensure a log event is written to the API logs but also that the event still write back to the appropriate endpoint.

### CODE REVIEW

File: app/controllers/v0/ppiu_controller.rb [hunk 1]
```diff
@@ -17,6 +17,7 @@ def index
 
     def update
       response = service.update_payment_information(pay_info)
+      Rails.logger.info('[PPIUUpdate]:requestcompleted', sso_logging_info)
```
Comment (bosawt) [inline app/controllers/v0/ppiu_controller.rb:20]: I think these should follow a format that looks more like this:
'PPIUController#update request completed'

**Comment by bosawt** [comment]
I tested this on localhost and this is an example of the resulting log:

> [PPIUUpdate]:requestcompleted -- { :user_uuid => "6400bbf301eb4e6e95ccea7693eced6f", :sso_cookie_contents => { "patientIcn" => "1008709396V637156", "mhvCorrelationId" => "4795335", "signIn" => { "serviceName" => "idme", "accountType" => "N/A" }, "credential_used" => "id_me", "expirationTime" => "2021-05-24T21:53:48Z" }, :request_host => "localhost" }

File: app/controllers/v0/profile/email_addresses_controller.rb [hunk 2]
```diff
@@ -29,6 +31,7 @@ def destroy
           add_effective_end_date(email_address_params),
           http_verb: 'put'
         )
+        Rails.logger.info('EmailAddressesController#destory request completed', sso_logging_info)
```
Comment (bosawt) [inline app/controllers/v0/profile/email_addresses_controller.rb:34]: Small typo here: `destory`

**Review by bosawt — APPROVED** [review]
Looks good! Just found one tiny typo

File: app/controllers/v0/profile/email_addresses_controller.rb (hunk 2 above)
Comment (joeniquette) [reply app/controllers/v0/profile/email_addresses_controller.rb]: thank you for catching this :)

**Review by thilton-oddball — APPROVED** [review]
Looks good!

**Pull request merged**